ultralight-java-databind/src/main/java/net/labymedia/ultralight/api/JavaAPI.java

```diff
@@ -0,0 +1,22 @@
+package net.labymedia.ultralight.api;
+
+import net.labymedia.ultralight.UltralightDatabind;
+import net.labymedia.ultralight.javascript.JavascriptClass;
+
+public final class JavaAPI {
+    private final UltralightDatabind databind;
+
+    public JavaAPI(UltralightDatabind databind) {
+        this.databind = databind;
+    }
+
+    public JavascriptClass importClass(String name) {
+        try {
+            return databind.toJavascript(Class.forName(name));
+        } catch (IllegalAccessException | ClassNotFoundException exception) {
+            System.out.println("About to shit myself");
+            exception.printStackTrace();
+            return null;
```

Comment:
This can just throw the exceptions, no need to handle them, they should be translated automatically.